src/modules/dashboard/components/Dialogs/GnosisControlSafeDialog/DetailsItem/DetailsItem.tsx

```diff
@@ -1,21 +1,30 @@
 import React from 'react';
 import { FormattedMessage, MessageDescriptor } from 'react-intl';
+import classnames from 'classnames';
 
 import styles from './DetailsItem.css';
 
 const DetailsItem = ({
   label,
   value,
+  isLastItem,
 }: {
   label: MessageDescriptor;
   value: JSX.Element;
-}) => (
-  <div className={styles.detailsItem}>
-    <div className={styles.detailsItemLabel}>
-      <FormattedMessage {...label} />
+  isLastItem?: boolean;
+}) => {
+  return (
+    <div
+      className={classnames(styles.detailsItem, {
+        [styles.separator]: !isLastItem,
+      })}
```

Comment:
Couldn't you have done a `.detailsItem:last-child` to select the element that doesn't need the border? That way you avoid the extra prop and logic